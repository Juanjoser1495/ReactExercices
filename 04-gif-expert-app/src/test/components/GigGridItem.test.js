import { shallow } from "enzyme";
import GifGridItem from "../../components/GifGridItem";

describe("Pruebas en <GifGridItem />", () => {
  const title = "A title";
  const url = "https://localhost/algo.jpg";
  test("Debe de mostrar el <GifGridItem> correctamnete", () => {
    let wrapper = shallow(<GifGridItem title={title} url={url}></GifGridItem>);
    expect(wrapper).toMatchSnapshot();
  });
});
