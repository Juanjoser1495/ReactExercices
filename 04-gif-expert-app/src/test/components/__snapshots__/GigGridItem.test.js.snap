// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> Debe de mostrar el <GifGridItem> correctamnete 1`] = `
<div
  className="card"
>
  <img
    alt="A title"
    src="https://localhost/algo.jpg"
  />
  <p>
    A title
  </p>
</div>
`;
